appdistribution/app/src/main/java/com/google/firebase/appdistributionquickstart/kotlin/ComposeMainActivity.kt

```diff
@@ -0,0 +1,114 @@
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Scaffold
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.material.TopAppBar
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.DisposableEffect
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalLifecycleOwner
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.Lifecycle
+import androidx.lifecycle.LifecycleEventObserver
+import androidx.lifecycle.LifecycleOwner
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.google.firebase.appdistributionquickstart.R
+import com.google.firebase.appdistributionquickstart.ui.theme.AppDistributionTheme
+
+
+class ComposeMainActivity : ComponentActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        setContent {
+            AppDistributionTheme {
+                Surface(
+                    modifier = Modifier.fillMaxSize(), color = MaterialTheme.colors.background
+                ) {
+                    MainAppView()
+                }
+            }
+        }
+    }
+}
+
+@Composable
+fun MainAppView(
+    appDistributionViewModel: AppDistributionViewModel = viewModel(factory = AppDistributionViewModel.Factory)
+) {
+
+    ComposableLifecycle { source, event ->
+        if (event == Lifecycle.Event.ON_RESUME) {
+            appDistributionViewModel.updateIfNewRelease()
+        }
+    }
+
+
+    Scaffold(topBar = {
+        TopAppBar(
+            backgroundColor = colorResource(R.color.colorPrimary)
+        ) {
+            Text(
+                text = stringResource(R.string.app_name),
+                style = MaterialTheme.typography.h6,
+                textAlign = TextAlign.Center,
+                modifier = Modifier.padding(8.dp),
+                color = Color.White
+            )
+        }
+    }, content = { it ->
+        Column(
+            horizontalAlignment = Alignment.CenterHorizontally, modifier = Modifier
+                .fillMaxSize()
+                .padding(it)
+        ) {
+            Spacer(modifier = Modifier.height(30.dp))
+            Image(
+                painterResource(R.drawable.firebase_lockup_400),
+                contentDescription = "",
```

Comment:
Can we add a content description to this image?
```suggestion
                contentDescription = "Firebase logo",
```